projects/plugins/jetpack/modules/contact-form/grunion-contact-form.php

```diff
@@ -168,17 +168,19 @@ protected function __construct() {
 		}
 
 		// custom post type we'll use to keep copies of the feedback items
+		$slug = 'edit.php?post_type=feedback';
```

Comment:
Since we only use it once in this function, maybe we can set the string directly in `add_menu_page`, without that new varialble?

Reply:
It's used in `register_post_type` too which is why I left that in.